db/seeds.rb

```diff
@@ -28,44 +28,56 @@
   license_type: Licenses.types.keys.first
 )
 
-site = Site.where(name: 'Prairie Center', business: business).first_or_create(
+montana = Lookup::State.find_by(abbr: 'MT')
```

Comment:
I know we're expecting these to be populated by the import but should these be `first_or_create`s just in case?